apps/okreads-e2e/src/integration/reading-list.spec.ts

```diff
@@ -11,4 +11,26 @@ describe('When: I use the reading list feature', () => {
       'My Reading List'
     );
   });
+
+  it('Then: I should be able to mark a book as finished', () => { 
```

Comment:
I would suggest to run application first, interact it with application so that you have few books in reading list and out of it 1-2 should be marked as finished. Then try to run E2E test case and see if all working fine or not.